openapi/components/schemas/ReportDeclinedTransactions.yaml

```
@@ -0,0 +1,28 @@
+type: object
+properties:
+  totalCount:
+    description: Total count of declined transactions.
```

nitpick: based on our style, we avoid using 'count' in the descriptions
```suggestion
    description: Total number of declined transactions.
```